feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -71,13 +71,13 @@ fun OfficeUI(
                 ),
                     color = MaterialTheme.colorScheme.tertiary,
                     textAlign = TextAlign.Justify,
-                    weight =FontWeight.Bold ,
+                    weight = FontWeight.Bold,
                     onClick = {})
                 LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
                     label = officeData.address,
                     color = MaterialTheme.colorScheme.onPrimaryContainer,
                     textAlign = TextAlign.Justify,
-                    weight = FontWeight.Normal ,
+                    weight = FontWeight.Normal,
                     onClick = {})
                 LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
                     label = stringResource(id = R.string.LPHours) + officeData.schedule,
```

Comment:
Instead of using a **+** after the **stringResource(id = R.string.LPHours)** you can add it as string resource parametter, check this for a better understanding.
![image](https://github.com/irv205/droid_leaspert/assets/33376944/5ad09c71-78b6-46fd-b9ff-a16d60bcab40)

Add it please, the same for Address label in stringResource =)